--- C:\NpackdSymlinks\com.nokia.QtDev-i686-w64-Npackd-Release-5.5.1\qtdeclarative\src\3rdparty\masm\yarr\YarrPattern_copy.h	2017-10-28 15:24:04.876136700 +0200
+++ C:\NpackdSymlinks\com.nokia.QtDev-i686-w64-Npackd-Release-5.5.1\qtdeclarative\src\3rdparty\masm\yarr\YarrPattern.h	2017-10-28 15:25:09.376136700 +0200
@@ -286,13 +286,13 @@
 // (please to be calling newlineCharacterClass() et al on your
 // friendly neighborhood YarrPattern instance to get nicely
 // cached copies).
-CharacterClass* newlineCreate();
-CharacterClass* digitsCreate();
-CharacterClass* spacesCreate();
-CharacterClass* wordcharCreate();
-CharacterClass* nondigitsCreate();
-CharacterClass* nonspacesCreate();
-CharacterClass* nonwordcharCreate();
+static CharacterClass* newlineCreate() { return 0;}
+static CharacterClass* digitsCreate() { return 0;}
+static CharacterClass* spacesCreate() { return 0;}
+static CharacterClass* wordcharCreate() { return 0;}
+static CharacterClass* nondigitsCreate() { return 0;}
+static CharacterClass* nonspacesCreate() { return 0;}
+static CharacterClass* nonwordcharCreate() { return 0;}
 
 struct TermChain {
     TermChain(PatternTerm term)
